nuxeo-features/rest-api/nuxeo-rest-api-server/src/main/java/org/nuxeo/ecm/restapi/server/jaxrs/usermanager/UserObject.java

```diff
@@ -59,9 +59,7 @@ protected void deleteArtifact() {
     }
 
     @Override
-    protected boolean isAPowerUserEditableArtifact() {
-        return UserRootObject.isAPowerUserEditableUser(currentArtifact);
-
+    protected boolean isAPowerUserEditableArtifact(NuxeoPrincipal artifact) {
+        return UserRootObject.isAPowerUserEditableUser(artifact);
     }
-
```

Comment:
can we keep it as it was before (no need to remove this line, perhaps you want it on master version)